stores/mongo.go

```diff
@@ -116,20 +116,20 @@ func (mongo *MongoStore) QueryBindings(service string, host string) ([]QBinding,
 	return qbindings, err
 }
 
-//InsertService inserts a new service into the datastore
-func (mongo *MongoStore) InsertService(name string, hosts []string, authTypes []string, authMethod string, retrievalField string, createdOn string) (QService, error) {
+//InsertServiceType inserts a new service into the datastore
+func (mongo *MongoStore) InsertServiceType(name string, hosts []string, authTypes []string, authMethod string, retrievalField string, createdOn string) (QServiceType, error) {
 
-	var qService QService
+	var qService QServiceType
 	var err error
 
-	qService = QService{Name: name, Hosts: hosts, AuthTypes: authTypes, AuthMethod: authMethod, RetrievalField: retrievalField, CreatedOn: createdOn}
+	qService = QServiceType{Name: name, Hosts: hosts, AuthTypes: authTypes, AuthMethod: authMethod, RetrievalField: retrievalField, CreatedOn: createdOn}
 	db := mongo.Session.DB(mongo.Database)
 	c := db.C("services")
```

Comment:
Refactoring resource name (services->service_types) should be reflected in data store as well for consistency

Reply:
Its been added in a later commit, but its true that it should be included here.